frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetAccessManagement/ClusterSetAccessManagement.tsx

```diff
@@ -456,7 +456,10 @@ function GroupUsersPopover(props: { group?: Group; useIcon?: boolean }) {
   }
   return (
     <div>
-      <Popover headerContent={t('access.usersInGroup')} bodyContent={<AcmLabels labels={props.group.users} />}>
+      <Popover
+        headerContent={t('access.usersInGroup')}
+        bodyContent={props.group.users ? <AcmLabels labels={props.group.users} /> : 'No users in group'}
```

Comment:
Hey Vishali - I just did a quick skim and have a couple things to note here:
1. I made a slight tweak to the YAML for `empty-test-group` and for that case I don't get the text 'No users in group'. 
2. The string 'No users in group' should be externalized for translation. You can simply change it to `t('No users in group')`, then run `npm run i18n:fix` from the `frontend` directory. This will add it to the translation file.

Reply:
updated, thanks for the feedback @KevinFCormier

Reply:
@vishsanghishetty Can you please update it so that the new message will also display if `props.group.users` is an empty array?

Reply:
yep, just updated

Reply:
Thank you!